### ISSUE TITLE
Got a crash when running a docker hello world example.

### ISSUE THREAD

**Issue by lonnblad** [mcollina/docker-loghose#1]
Got a crash when running a docker hello world example.

terminal 1.

```
sudo docker run --rm -v /var/run/docker.sock:/var/run/docker.sock matteocollina/docker-loghose

> docker-loghose@0.3.1 start /usr/src/app
> ./loghose.js


/usr/src/app/node_modules/never-ending-stream/never-ending-stream.js:14
    stream.destroy()
           ^
TypeError: Cannot call method 'destroy' of null
    at DestroyableTransform.result.destroy (/usr/src/app/node_modules/never-ending-stream/never-ending-stream.js:14:12)
    at detachContainer (/usr/src/app/loghose.js:31:19)
    at EventEmitter.<anonymous> (/usr/src/app/loghose.js:26:5)
    at EventEmitter.emit (events.js:98:17)
    at DestroyableTransform._transform (/usr/src/app/node_modules/docker-allcontainers/allcontainers.js:62:16)
    at DestroyableTransform.Transform._read (/usr/src/app/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (/usr/src/app/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/usr/src/app/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at writeOrBuffer (/usr/src/app/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:227:5)
    at DestroyableTransform.Writable.write (/usr/src/app/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:194:11)

npm ERR! Linux 3.19.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.10.36
npm ERR! npm  v2.5.0
npm ERR! code ELIFECYCLE
npm ERR! docker-loghose@0.3.1 start: `./loghose.js`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the docker-loghose@0.3.1 start script './loghose.js'.
npm ERR! This is most likely a problem with the docker-loghose package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./loghose.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls docker-loghose
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/src/app/npm-debug.log
```

terminal 2.

```
sudo docker run ubuntu:14.04 /bin/echo 'Hello world'
Hello world
```

BR
/ Fredrik

**Comment by mcollina** [comment]
I cannot reproduce, but I think I fixed the bug on never-ending-stream. Can you please verify?

**Comment by mcollina** [comment]
This should be fixed as of 0.8.0.
